LeagueInformer/LeagueInformer/Program.cs

```diff
@@ -145,7 +146,32 @@ private static async Task GetBestChallengers()
 
         private static async Task GetServerStatus()
         {
-            var response = await ServerService.GetServerStatus("eune");
+            Console.WriteLine(AppResources.GetServerStatus_ChooseServerFromList, Environment.NewLine, Environment.NewLine);
+            var position = 1;
+            foreach (var serverName in AppSettings.ServerAddresses.Keys)
+            {
+                Console.WriteLine(AppResources.GetServerStatus_PrintServersList,
+                    position,
+                    serverName);
```

Comment:
nie lepiej dać to w jednej linijce?

Reply:
Według zasad czystego kodu "Funkcje nie powinny mieć 100 wierszy długości. Funkcje powinny mieć właśnie nie więcej niż 20 wierszy"